test/e2e/specs/demo-content-with-keyboard.test.js

```diff
@@ -0,0 +1,345 @@
+/**
+ * Node dependencies
+ */
+import path from 'path';
+import fs from 'fs';
+import os from 'os';
+import uuid from 'uuid/v4';
+
+/**
+ * Internal dependencies
+ */
+import {
+	newPost,
+	pressWithModifier,
+	META_KEY,
+	pressTimes,
+	//getEditedPostContent,
+} from '../support/utils';
+
+async function uploadImageInTheMediaLibrary( assetFileName ) {
+	await page.waitForSelector( '.media-modal input[type=file]' );
+	const inputElement = await page.$( '.media-modal input[type=file]' );
+	const testImagePath = path.join( __dirname, '..', 'assets', assetFileName );
+	const filename = uuid();
+	const tmpFileName = path.join( os.tmpdir(), filename + '.png' );
+	fs.copyFileSync( testImagePath, tmpFileName );
+	await inputElement.uploadFile( tmpFileName );
+	await page.waitForSelector( `.media-modal li[aria-label="${ filename }"]` );
+}
+
+async function navigateToNextRegion() {
+	await page.keyboard.down( 'Alt' );
+	await page.keyboard.down( 'Shift' );
+	await page.keyboard.type( 'n' );
+	await page.keyboard.up( 'Shift' );
+	await page.keyboard.up( 'Alt' );
+}
+
+async function createBlockAfter() {
+	await page.keyboard.down( META_KEY );
+	await page.keyboard.down( 'Alt' );
+	await page.keyboard.type( 'y' );
+	await page.keyboard.up( 'Alt' );
+	await page.keyboard.up( META_KEY );
+}
+
+jest.setTimeout( 1000000 );
+
+describe( 'Demo content post', () => {
+	beforeAll( async () => {
```

Comment:
We should be using `beforeEach`, not `beforeAll`. Otherwise the editor state remains between test cases which is confusing/unexpected when you go to add a test, and if relied on really dangerous because it means our test cases aren't isolated.

Reply:
This should still be changed to `beforeEach`:

```suggestion
	beforeEach( async () => {
```